ishoborabyose/MyDiary
# Ft get entry by id 169621498

What does this PR do?
- This pull request is for creating entry Diary
- Modify entry
- get all entries
- get an entry by id
- delete entries
- tests

Description of Task to be completed?
- insert codes in diary controller
- insert codes into middleware
- insert codes into test file
- insert into json


How should this be manually tested?
- Clone the repo
- Cd into the project folder folder
- npm start
- npm test
Any background context you want to provide?
Not Applicable
